components/ButtonGenerator/CodeBlock.tsx

```diff
@@ -115,12 +125,67 @@ export default function CodeBlock({ button }): JSX.Element {
     }
   };
 
+  const checkReactValue = (value, attribute) => {
+    if (value !== undefined && value !== '') {
+      if (attribute === 'currency') {
+        if (['USD', 'CAD', 'EUR', 'GBP', 'AUD'].includes(value)) {
```

Comment:
I'd rather this come from a central list somewhere. Maybe from the DB. It'll be one of those things we miss when we add/remove currencies. In fact EUR, GPB, and AUD aren't even supported rn.